src/Uno.UI/UI/Xaml/Controls/TimePicker/TimePickerFlyout.Android.cs

```diff
@@ -61,25 +65,35 @@ private void ShowUsingAlertDialog()
 		{
 			//On Samsung devices ?= 6.0 the TimepickerDialog is not displayed properly in Landscape.
 			//Instead using this work around seems to work.
-			var androidTimePicker = new Android.Widget.TimePicker(ContextHelper.Current);
+			var timePicker = new Android.Widget.TimePicker(ContextHelper.Current);
 
 			//Hour and Minute properties are not supported on Android before 6.0
-			androidTimePicker.Hour = Time.Hours;
-			androidTimePicker.Minute = Time.Minutes;
-
-			androidTimePicker.SetIs24HourView(new Java.Lang.Boolean(ClockIdentifier == ClockIdentifiers.TwentyFourHour));
+			if (Build.VERSION.SdkInt >= BuildVersionCodes.M)
+			{
+				timePicker.Hour = Time.Hours;
+				timePicker.Minute = Time.Minutes;
+			}
+			else
+			{
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Which member is supposed to be used instead?

Reply:
if (Build.VERSION.SdkInt >= BuildVersionCodes.M)
			{
				timePicker.Hour = Time.Hours;
				timePicker.Minute = Time.Minutes;
			}
			else
			{
#pragma warning disable CS0618 // Type or member is obsolete
				timePicker.CurrentHour = new Java.Lang.Integer(Time.Hours);
				timePicker.CurrentMinute = new Java.Lang.Integer(Time.Minutes);
#pragma warning restore CS0618 // Type or member is obsolete
			}